Tests/resources/OrderHelperData.php

```diff
@@ -0,0 +1,229 @@
+                'OXCITY',
+                'OXCOUNTRYID',
+                'OXSTATEID',
+            ],
+            'rows' => [
+                [
+                    'User ID 1',
+                    1,
+                    'User 1',
+                    'User',
+                    'One',
+                    'Red Company',
+                    'Blue Square',
+                    '1',
+                    '5555',
+                    'Green City',
+                    '1',
+                    '1'
+                ],
+                [
+                    'User ID 2',
+                    1,
+                    'User 2',
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null,
+                    null
+                ],
+            ],
+        ],
+        [
+            'table' => 'oxaddress',
+            'columns' => [
+                'OXID',
+                'OXUSERID',
+                'OXFNAME',
+                'OXLNAME',
+                'OXCOMPANY',
+                'OXSTREET',
+                'OXSTREETNR',
+                'OXZIP',
+                'OXCITY',
+                'OXCOUNTRYID',
+                'OXSTATEID',
+            ],
+            'rows' => [
+                [
+                    '1',
+                    'User ID 1',
+                    'First',
+                    'Last',
+                    'Company',
+                    'Street',
+                    '10',
+                    '9876',
+                    'City',
+                    '9',
+                    '9',
+                ],
+            ],
+        ],
+        [
+            'table' => 'oxcountry',
+            'columns' => [
+                'OXID',
+                'OXTITLE',
+            ],
+            'rows' => [
+                [
+                    '9',
+                    'Testland',
+                ],
+            ],
+        ],
+        [
+            'table' => 'oxorder',
+            'columns' => [
+                'OXID',
+                'OXUSERID',
+                'OXORDERDATE',
+                'OXDELFNAME',
+                'OXDELLNAME',
+                'OXDELCOMPANY',
+                'OXDELSTREET',
+                'OXDELSTREETNR',
+                'OXDELZIP',
+                'OXDELCITY',
+                'OXDELCOUNTRYID',
+                'OXDELSTATEID',
+                'OXBILLFNAME',
+                'OXBILLLNAME',
+                'OXBILLCOMPANY',
+                'OXBILLSTREET',
+                'OXBILLSTREETNR',
+                'OXBILLZIP',
+                'OXBILLCITY',
+                'OXBILLCOUNTRYID',
+                'OXBILLSTATEID',
+            ],
+            'rows' => [
+                [
+                    1,
+                    'User ID 1',
+                    '2019-03-30 10:55:13',
+                    'Oxid',
```

Comment:
I would suggest making use of the `TEST_USER_1` data (starting from 'Oxid') as it makes the correlation more obvious.